apps/token-manager/app/src/App.js

```diff
@@ -27,11 +29,17 @@ class App extends React.Component {
     appStateReady: false,
     holders: [],
     userAccount: '',
+    groupMode: false,
   }
   state = {
     assignTokensConfig: initialAssignTokensConfig,
     sidepanelOpened: false,
   }
+  getHolderBalance(address) {
+    const { holders } = this.props
+    const holder = holders.find(holder => holder.address === address)
```

Comment:
This should use `addressesEqual()`, otherwise the comparison will fail between checksummed and non-checksummed (resulting in odd amounts, like `2` when they should be `0`).